.github/workflows/repo-health-job.yml

```diff
@@ -78,6 +78,11 @@ jobs:
         run: |
           python3 -m pip install -r edx-repo-health/requirements/pip.txt
           python3 -m pip install -r edx-repo-health/requirements/pip-tools.txt
+
+      - name: Install edx-repo-health requirements
```

Comment:
Won't this be done later by https://github.com/openedx/edx-repo-health/blob/master/scripts/repo-health-script.sh#L36 anyway?

Reply:
I included this additional step because we've removed this step from the artifact script. Removing this additional step since it'll probably already be present inside the environment as highlighted.